src/lib/gssapi/mechglue/g_imp_sec_context.c

```diff
@@ -97,7 +97,7 @@ gss_ctx_id_t *		context_handle;
     /* Initial value needed below. */
     status = GSS_S_FAILURE;
 
-    ctx = (gss_union_ctx_id_t) malloc(sizeof(gss_union_ctx_id_desc));
+    ctx = (gss_union_ctx_id_t)calloc(sizeof(gss_union_ctx_id_desc), 1);
```

Comment:
IIRC the MIT style guide requires a space after a return value cast?  Or am I confusing it with the Illumos style guide?

Also, the arguments of `calloc()` can be in either order, but the first one is the one called `nmemb`, so the first one ought to be `1`, but this is a nit I don't usually pick and frankly idk why I'm picking it here!  :)